src/System.ServiceModel.Tests.Common/src/BridgeTestFixture.cs

```diff
@@ -0,0 +1,148 @@
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+
+namespace System.ServiceModel.Tests.Common
+{
+    public static class BridgeTestFixture 
+    {
+        private static Dictionary<string, string> _Resources = new Dictionary<string, string>();
+        private static BridgeState _BridgeStatus = BridgeState.NotStarted;
+        private static Dictionary<string, string> _BaseAddresses = null;
+
+        private static string BridgeBaseAddress
+        {
+            // TODO: Pull this address from msbuild props, env vars, or config passed into xunit.
+            get { return "http://localhost:44283"; }
```

Comment:
Go ahead and leave this hard-coded -- I'll make the change to use TestProperties after you check-in